bounties/php/CMS/1/vulnerability.json

```diff
@@ -0,0 +1,52 @@
+{
+    "PackageVulnerabilityID": "1-Lack of rate limiting",
```

Comment:
**`PackageVulnerabilityID` is the order of disclosure on a particular target. In this case, this is the first disclosure on this project on huntr**

So change it to:

```json
"PackageVulnerabilityID": "1-Lack of rate limiting"
```

**to** :point_down: 

```json
"PackageVulnerabilityID": 1
```